### PR TITLE
[Docs] Alexwangxiang/t-sne_visualize_doc

### PR DESCRIPTION
**Pull request by alexwangxiang**
## Modification

Add Chinese documentation for t-SNE visualization tool.

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/useful_tools/t-sne_visualization.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# t-Distributed Stochastic Neighbor Embedding (t-SNE) Visualization
+
+## Introduction of the t-SNE visualization tool
+
+MMPretrain provides `tools/visualization/vis_tsne.py` tool to visualize ... by t-SNE. Please install `sklearn` to calculate TSNE by `pip install scikit-learn`.
```
Comment (fangyixiao18) [inline docs/en/useful_tools/t-sne_visualization.md:5]:
```suggestion
MMPretrain provides `tools/visualization/vis_tsne.py` tool to visualize high-dimensional data vectors by t-SNE. Please install `sklearn` to calculate TSNE by `pip install scikit-learn`.
```

File: docs/en/useful_tools/t-sne_visualization.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    --checkpoint https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_b16x8_cifar10_20210528-f54bfad9.pth
+```
+
+| ResNet-18                                                                                            | ResNet-50                                                                                            |
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| <div align=center><img src='https://user-images.githubusercontent.com/42371271/236410521-c4d087da-d16f-48ad-b951-c74d10c68f33.png' height="auto" width="auto" ></div> | <div align=center><img src='https://user-images.githubusercontent.com/42371271/236411844-c97dc514-dad0-401e-ba8f-307d0a385b4e.png' height="auto" width="auto" ></div> |
+
+## How to visualize the t-SNE of a self-supervised vision transformer
+
+To be added.
```
Comment (fangyixiao18) [inline docs/en/useful_tools/t-sne_visualization.md:74]: could you please update this part?

File: docs/zh_CN/useful_tools/t-sne_visualization.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    --checkpoint https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_b16x8_cifar10_20210528-f54bfad9.pth
+```
+
+| ResNet-18                                                                                            | ResNet-50                                                                                            |
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| <div align=center><img src='https://user-images.githubusercontent.com/42371271/236410521-c4d087da-d16f-48ad-b951-c74d10c68f33.png' height="auto" width="auto" ></div> | <div align=center><img src='https://user-images.githubusercontent.com/42371271/236411844-c97dc514-dad0-401e-ba8f-307d0a385b4e.png' height="auto" width="auto" ></div> |
+
+## 如何可视化自监督视觉 transformer 的t-分布随机邻域嵌入
+
+待添加。
```
Comment (fangyixiao18) [inline docs/zh_CN/useful_tools/t-sne_visualization.md:74]: could you please also update this part?

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/zh_CN/useful_tools/t-sne_visualization.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    --checkpoint https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_b16x8_cifar10_20210528-f54bfad9.pth
+```
+
+| ResNet-18                                                                                            | ResNet-50                                                                                            |
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| <div align=center><img src='https://user-images.githubusercontent.com/42371271/236410521-c4d087da-d16f-48ad-b951-c74d10c68f33.png' height="auto" width="auto" ></div> | <div align=center><img src='https://user-images.githubusercontent.com/42371271/236411844-c97dc514-dad0-401e-ba8f-307d0a385b4e.png' height="auto" width="auto" ></div> |
+
+## 如何可视化自监督视觉 transformer 的t-分布随机邻域嵌入
+
+以下是在ImageNet数据集上训练的 MAE-ViT-base 模型上运行 t-SNE 可视化的一个样例。输入数据来自 ImageNet 验证集。
```
Comment (Ezra-Yu) [inline docs/zh_CN/useful_tools/t-sne_visualization.md:74]: It needs to be explained here that MAE and some self-supervised pre-training algorithms do not have test_dataloader information. When analyzing such self-supervised algorithms, you need to add test_dataloader information in the config, or you can use '--test-cfg' to specify a file

File: docs/zh_CN/useful_tools/t-sne_visualization.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+- `--show`: 在图形窗口中显示结果。
+- `--n-components N_COMPONENTS`: 结果的维数。
+- `--perplexity PERPLEXITY`: 复杂度与其他流形学习算法中使用的最近邻的数量有关。
+- `--early-exaggeration EARLY_EXAGGERATION`: 控制原空间中的自然聚类在嵌入空间中的紧密程度以及它们之间的空间大小。
+- `--learning-rate LEARNING_RATE`: t-SNE 的学习率通常在[10.0, 1000.0]的范围内。如果学习率太高，数据可能看起来像一个球，其中任何一点与它最近的邻居近似等距。如果学习率太低，大多数点可能看起来被压缩在一个几乎没有异常值的密集点云中。
+- `--n-iter N_ITER`: 优化的最大迭代次数。应该至少为250。
+- `--n-iter-without-progress N_ITER_WITHOUT_PROGRESS`: 在我们中止优化之前，最大的没有进展的迭代次数。
+- `--init INIT`: 初始化方法。
+
+## 如何可视化 CNN 分类器的t-分布随机邻域嵌入（如 ResNet-18 和 ResNet-50）
```
Comment (Ezra-Yu) [inline docs/zh_CN/useful_tools/t-sne_visualization.md:54]: ## 如何可视化分类算法的 t-SNE  (如 ResNet)

File: docs/zh_CN/useful_tools/t-sne_visualization.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+python tools/visualization/vis_tsne.py \
+    configs/resnet/resnet50_8xb16_cifar10.py \
+    --checkpoint https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_b16x8_cifar10_20210528-f54bfad9.pth
+```
+
+| ResNet-18                                                                                            | ResNet-50                                                                                            |
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| <div align=center><img src='https://user-images.githubusercontent.com/42371271/236410521-c4d087da-d16f-48ad-b951-c74d10c68f33.png' height="auto" width="auto" ></div> | <div align=center><img src='https://user-images.githubusercontent.com/42371271/236411844-c97dc514-dad0-401e-ba8f-307d0a385b4e.png' height="auto" width="auto" ></div> |
+
+## 如何可视化自监督视觉 transformer 的t-分布随机邻域嵌入
```
Comment (Ezra-Yu) [inline docs/zh_CN/useful_tools/t-sne_visualization.md:72]: ## 如何可视化自监督算法的 t-SNE (如 MAE)

File: docs/zh_CN/useful_tools/t-sne_visualization.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+- `CONFIG`: t-SNE 配置文件的路径。
+- `--checkpoint CHECKPOINT`: 模型权重文件的路径。
+- `--work-dir WORK_DIR`: 保存日志和可视化图像的目录。
+- `--test-cfg TEST_CFG`: 用来加载 test_dataloader 配置的 t-SNE 配置文件的路径。
+- `--vis-stage {backbone,neck,pre_logits}`: 模型可视化的阶段。
+- `--class-idx CLASS_IDX [CLASS_IDX ...]`: 用来计算 t-SNE 的类别。
+- `--max-num-class MAX_NUM_CLASS`: 前 N 个被应用 t-SNE 算法的类别，默认为20。
+- `--max-num-samples MAX_NUM_SAMPLES`: 每个类别中最大的样本数，值越高需要的计算时间越长，默认为100。
+- `--cfg-options CFG_OPTIONS [CFG_OPTIONS ...]`: 覆盖被使用的配置中的一些设定，xxx=yyy 格式的关键字-值对会被合并到配置文件中。override some settings in the used config, the key-value pair in xxx=yyy format will be merged into config file. If the value to be overwritten is a list, it should be like key="[a,b]" or key=a,b It also allows nested list/tuple values, e.g. key="[(a,b),(c,d)]" Note that the quotation marks are necessary and that no white space is allowed.
```
Comment (Ezra-Yu) [inline docs/zh_CN/useful_tools/t-sne_visualization.md:42]: 这里很多行没有翻译

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**